### DISCUSSION TITLE
Xamarin Forms Configuring Services

### DISCUSSION THREAD

**Discussion by bb281** [shinyorg/shiny#1452] [Q&A]
Is there any documentation for supporting Xamarin Forms with the latest 3.X library?

I'm trying to figured out where I wireup and configure my services so I can setup a HTTP Transfer in Xamarin Forms. 

Previously, there was ShinyStartup class where you can override the ConfigureServices, but I believe that is gone now. Appreciate any pointers.

**Comment by aritchie** [comment]
https://shinylib.net/client/hosting/manual/
